x/datadeal/keeper/deal.go

```diff
@@ -14,6 +15,11 @@ import (
 	oracletypes "github.com/medibloc/panacea-core/v2/x/oracle/types"
 )
 
+const (
+	BlockPeriod           = 6 * time.Second
```

Comment:
I suggest to make the `blockTimeSec` in `cmd/init.go` public and use it?

Reply:
Okay, thanks. I change to use it.

Reply:
When I checked because the CI test failed, an import cycle error occurred in that case. So, I reverted the commit. I think it looks good to keep it this way.